lib/crowbar/client/command/proposal/create.rb

```diff
@@ -103,6 +103,10 @@ def from_file
             rescue
               err "Failed to process file"
             end
+            args.proposal = args.proposal || json["id"]
+            if json["id"] != args.proposal
+              err "id(#{json["id"]}) in file and name(#{args.proposal}) should be same"
```

Comment:
In case the id and the arg is not the same I would just use the id and give a warning.